### DISCUSSION TITLE
LiveStream error on v12 beta

### DISCUSSION THREAD

**Discussion by DerMuri** [dgreif/ring#1287] [Q&A]
Hi everyone.
I'm new to Homebridge and I'm setting everything up.
When trying to start a livestream on my Ring doorbell I get the following error message.

Is it an issue with the beta plugin or is the problem on my side?

Best regards
Muri

<img width="1167" alt="Bildschirmfoto 2023-08-09 um 09 54 58" src="https://github.com/dgreif/ring/assets/25777499/8197cd68-2357-4e58-95a9-8b2e6a2aa488">

**Comment by robbiem36** [comment]
I'm having the same issue.

**Comment by smockle** [comment]
@valerioceniccola reported the same “RangeError: offset is out of bounds” issue in https://github.com/dgreif/ring/issues/1253#issuecomment-1669165196.

**Reply by smockle** [reply]
https://github.com/dgreif/ring/issues/1286 is another report.
